test/extended/cli/mustgather.go

```diff
@@ -337,6 +337,50 @@ var _ = g.Describe("[sig-cli] oc adm must-gather", func() {
 			})
 		})
 	})
+
+	g.It("runs successfully for metrics gathering", func() {
+		tempDir, err := os.MkdirTemp("", "test.oc-adm-must-gather.")
+		o.Expect(err).NotTo(o.HaveOccurred())
+		defer os.RemoveAll(tempDir)
+
+		args := []string{
+			"--dest-dir", tempDir,
+			"--",
+			"/usr/bin/gather_metrics",
+			fmt.Sprintf("--min-time=%d", time.Now().Add(-5*time.Minute).UnixMilli()),
+			fmt.Sprintf("--max-time=%d", time.Now().UnixMilli()),
+			"--match=prometheus_ready",
+			"--match=prometheus_build_info",
+		}
+		o.Expect(oc.Run("adm", "must-gather").Args(args...).Execute()).To(o.Succeed())
+
+		// wait for the contents to show up in the plugin output directory
+		time.Sleep(5 * time.Second)
```

Comment:
This might not be very robust but I see that the same pattern is used in the other tests.